### PR TITLE
Demo portal

### PR DESCRIPTION
**Pull request by mmadariaga**
Added a demo portal within the project

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+  <title>Ivoz Provider - Client Administation portal</title>
```
Comment (fmendozzac) [inline demo/index.html:31]: Administration

File: demo/src/entities/Client/Client.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+import { foreignKeyGetter } from './ForeignKeyGetter';
+import { ClientProperties } from './ClientProperties';
+import foreignKeyResolver from './ForeignKeyResolver';
+import RemoteId from './Field/RemoteId';
+import Actions from './Action';
+
+const properties: ClientProperties = {
+  iden: {
+    label: _('Name'),
+    pattern: new RegExp('^[^\\s]+$'),
```
Comment (fmendozzac) [inline demo/src/entities/Client/Client.tsx:16]: better / instead '

File: demo/src/entities/Client/Client.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+};
+
+const client: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: ApartmentIcon,
+  iden: 'Client',
+  title: 'Clients',
+  path: '/clients',
+  defaultOrderBy: '',
+  toStr: (row: any) => row.iden,
```
Comment (fmendozzac) [inline demo/src/entities/Client/Client.tsx:113]: any is forbidden

File: demo/src/entities/Client/Field/RemoteId.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  >([]);
+
+  useEffect(() => {
+    if (!platformId) {
+      return;
+    }
+
+    RemoteIdSelectOptions(
+      {
+        callback: (options: any, rows: any) => {
```
Comment (fmendozzac) [inline demo/src/entities/Client/Field/RemoteId.tsx:40]: any is forbidden

File: demo/src/entities/Client/SelectOptions/SelectOptions.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import client from '../Client';
+
+const ClientSelectOptions: SelectOptionsType = ({
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  return defaultEntityBehavior.fetchFks(
+    client.path,
+    ['id', 'iden'],
+    (data: any) => {
```
Comment (fmendozzac) [inline demo/src/entities/Client/SelectOptions/SelectOptions.ts:13]: any is forbidden

File: demo/src/entities/Platform/Platform.tsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+};
+
+const platform: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: CastleIcon,
+  iden: 'Platform',
+  title: 'Platforms',
+  path: '/platforms',
+  defaultOrderBy: '',
+  toStr: (row: any) => row.name,
```
Comment (fmendozzac) [inline demo/src/entities/Platform/Platform.tsx:72]: any is forbidden

File: demo/src/entities/User/User.tsx [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+import foreignKeyResolver from './ForeignKeyResolver';
+import Form from './Form';
+import { UserProperties } from './UserProperties';
+import Actions from './Action';
+
+const properties: UserProperties = {
+  iden: {
+    label: _('Name'),
+    required: true,
+    pattern: new RegExp('^[^\\s]+$'),
```
Comment (fmendozzac) [inline demo/src/entities/User/User.tsx:20]: use / instead '

File: demo/src/App.tsx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+import { languagesList } from './translations/languages';
+
+export default function App(): JSX.Element {
+  StoreContainer.store = store;
+  const setLanguages = useStoreActions(
+    (actions: any) => actions.i18n.setLanguages
+  );
+  setLanguages(languagesList);
+
+  const apiSpecStore = useStoreActions((actions: any) => {
```
Comment (fmendozzac) [inline demo/src/App.tsx:19]: any is forbidden

File: demo/src/entities/Client/Client.tsx (hunk 2 above)
Comment (mmadariaga) [reply demo/src/entities/Client/Client.tsx:16]: it seems to work fine

File: demo/src/entities/User/User.tsx (hunk 7 above)
Comment (mmadariaga) [reply demo/src/entities/User/User.tsx:20]: It seems to be working fine

**Review by fmendozzac — APPROVED** [review]

**Pull request merged**